rotkehlchen/chain/ethereum/balancer/typing.py

```diff
@@ -0,0 +1,74 @@
+from typing import (
+    Any,
+    DefaultDict,
+    Dict,
+    List,
+    NamedTuple, NewType,
+    Optional,
+    Set,
+)
+
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.constants import ZERO
+from rotkehlchen.fval import FVal
+from rotkehlchen.typing import ChecksumEthAddress
+
+
+# Below set of classes where to store PoolShare, Pool and PoolToken data
+# BalancerPoolAsset stores PoolToken data, and adds custom ones
+class BalancerPoolAsset(NamedTuple):
+    address: ChecksumEthAddress  # PoolToken.id, token contract address
+    balance: FVal  # PoolToken.balance
+    denorm_weight: FVal  # PoolToken.denormWeight
+    name: str  # PoolToken.name
+    symbol: str  # PoolToken.symbol
+    # Custom fields
+    user_balance: FVal  # Estimated token balance
```

Comment:
for `user_balance` + `user_balance_usd` you can combine them using our [Balance](https://github.com/rotki/rotki/blob/af926e58a03ad187491259a286919be83dd26e91/rotkehlchen/accounting/structures.py#L71) data structure

Reply:
Cool, I saw the class but wasn't sure; this repo has plenty of classes!

Reply:
done